src/EditorFeatures/Core/Implementation/IntelliSense/AsyncCompletion/ItemManager.cs

```diff
@@ -176,8 +174,23 @@ internal ItemManager(RecentItemsManager recentItemsManager)
 
             // Use a monotonically increasing integer to keep track the original alphabetical order of each item.
             var currentIndex = 0;
+
             var builder = ArrayBuilder<MatchResult>.GetInstance();
```

Comment:
```suggestion
            using var _ = ArrayBuilder<MatchResult>.GetInstance(out var builder);
```

(and change ToImmutableAndFree to ToImmmutable)